cookbook/doctrine/registration_form.rst

```diff
@@ -19,12 +19,12 @@ first start with :doc:`/cookbook/security/entity_provider`.
 
 Your ``User`` entity will probably at least have the following fields:
 
-``username``
+* ``username``
```

Comment:
can you please explain why you did this? The previous definition list syntax seems better than the unordered list syntax.